recipes/oracle-instant-client/activate.sh

```diff
@@ -0,0 +1,29 @@
+pkgver="@PKG_VERSION@"
```

Comment:
Is there any reason for doing this in an activation script instead of post-link?

Reply:
Yeah I debated. Doing this on activation let's people rety the download and link as opposed to getting once chance with the post-link. OTOH post-link is the usual pattern for this kind of code.